src/showAPopUpScreen.ts

```diff
@@ -0,0 +1,16 @@
+export const showAPopUpScreen = (screenToDisplay: HTMLDivElement, startingPosition: string) => {
```

Comment:
I don't understand, why you need to pass "startingPosition" as string/'flex', why not just hardcode it there?
Do you plan to use this function elsewhere with different parameter?

Reply:
also, unifying names? `showAModalScreen` instead of `PopUp`

Reply:
also! I see a problem with this function. Switch to polish, easier to explain :) 
Długo myślałem co mi nie siedzi w tej funkcji - po co musimy tak dodawać tego event listenera? Dodawać i usuwać, dodawać i usuwać... I w końcu wykminiłem!
Bo ta funkcja robi wiele rzeczy - zamiast robić jedną.
samo dodawanie eventListenera do exitButtona powinno być wewnątrz funkcji `renderInfoModal` - u nas "render" oznacza wstępne zasetupowanie.
W ten sposób będziemy mieli proste dwa buttony:
`howToPlayButton` będzie miał eventListenera: `() =>      screenToDisplay.style.display = 'flex'`
`extiButton` będzie miał eventListenera: `() =>      screenToDisplay.style.display = 'none'`
i tyle, nic więcej nie będzie potrzebne w tych event listenerach :D